taker-frontend/src/App.tsx

```diff
@@ -169,68 +175,92 @@ export const App = () => {
-                                path="wallet"
-                                element={
-                                    <>
-                                        <Wallet walletInfo={walletInfo} />
-                                    </>
-                                }
-                            />
-                            <Route
-                                element={
-                                    // @ts-ignore: ts-lint thinks that {children} is missing but react router is taking care of this for us
-
-
-                                        <PageLayout
-                                            cfds={cfds}
-                                            connectedToMaker={connectedToMaker}
-                                            showPromoBanner={isWithinPromoPeriod}
-                                        />
-
-                                }
+            <Box
+                bg={useColorModeValue("white", "black")}
+                bgGradient={useColorModeValue(
+                    "linear(to-r, white 20%, gray.800, white 80%)",
+                    "linear(to-r, gray.800 20%, white, gray.800 80%)",
+                )}
+            >
+                <Center>
+                    <Box
+                        maxWidth={(VIEWPORT_WIDTH + 400) + "px"}
+                        width={"100%"}
+                        height={"100%"}
+                        bg={useColorModeValue("gray.200", "gray.700")}
+                    >
+                        <Center>
+                            <Box
+                                textAlign="center"
+                                padding={3}
+                                bg={useColorModeValue(BG_LIGHT, BG_DARK)}
+                                maxWidth={VIEWPORT_WIDTH_PX}
+                                width={"100%"}
+                                marginTop={"100px"}
                             >
-                                <Route
-                                    path="long"
-                                    element={
-                                        <>
-                                            <Trade
-                                                offer={longOffer}
-                                                connectedToMaker={connectedToMaker}
-                                                walletBalance={walletInfo ? walletInfo.balance : 0}
-                                                isLong={true}
+                                <Routes>
+                                    <Route path="/">
+                                        <Route
+                                            path="wallet"
+                                            element={
+                                                <>
+                                                    <Wallet walletInfo={walletInfo} />
+                                                </>
+                                            }
```

Comment:
The fragment here seems redundant.

```suggestion
                                            element={<Wallet walletInfo={walletInfo} />}
```